File: code-coaches/medium/initials/Initials.cs
using System;

class Initials
{
    static string GetInitials(string name)
    {
        string[] nameSplit = name.Split();
        return $"{nameSplit[0][0]}{nameSplit[1][0]}";
    }

    static void Main(string[] args)
    {
        int numNames = int.Parse(Console.ReadLine());
        string[] names = new string[numNames];
        for (int i = 0; i < numNames; i++)
        {
            names[i] = GetInitials(Console.ReadLine());
        }

        Console.WriteLine(string.Join(" ", names));
    }
}

File: code-coaches/medium/initials/Initials.swift
func getInitials(name: String) -> String {
    let nameSplit = name.split(separator: " ")
    return "\(nameSplit[0].prefix(1))\(nameSplit[1].prefix(1))"
}

func main() {
    let numNames = Int(readLine()!)!

	var names = [String]()
	for _ in 0..<numNames {
		let name = readLine()!
		let initials = getInitials(name: name)
		names.append(initials)
	}

	print(names.joined(separator: " "))
}

main()